src/window/mouse_manager.rs

```diff
@@ -60,6 +60,11 @@ fn mouse_button_to_button_text(mouse_button: &MouseButton) -> Option<String> {
     }
 }
 
+struct DragDetails {
+    draw_details: WindowDrawDetails,
+    button: String,
```

Comment:
Given that this can only be a static set of options, maybe this could be an enum?

Reply:
I changed it to the winit `MouseButton` enum, at the cost of an additional string conversion each time the drag message is sent.